tests/scenarios/PowershellScriptTest/config.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <applications>
+        <application>
+            <id>PSNoScripts</id>
+            <executable>PowershellScriptTest.exe</executable>
+        </application>
+      <application>
+        <id>PSOnlyStart</id>
+        <executable>PowershellScriptTest.exe</executable>
+        <startScript>
+          <scriptPath>SayHello.ps1</scriptPath>
+          <runOnce>false</runOnce>
+        </startScript>
+      </application>
+      <application>
+        <id>PSBothStartingFirst</id>
+        <executable>PowershellScriptTest.exe</executable>
+        <startScript>
+          <scriptPath>SayHello.ps1</scriptPath>
+          <runOnce>false</runOnce>
+        </startScript>
+        <endScript>
+          <scriptPath>SayHi.ps1</scriptPath>
+          <runOnce>false</runOnce>
+        </endScript>
+      </application>
+      <!-- I am omitting PSBothEndingFirst because the xsl file will always put
```

Comment:
>PSBothEndingFirst [](start = 25, length = 17)

Can you explain this comment?

Reply:
I wanted to make sure that the starting script and ending script would run regardless of if the ending script came before the starting script in the configuration.

I can't run that test when converting config.xml to config.json because the xsl always puts the starting script before the ending script.  Because I can't change the order I removed that test from the xml configuration.

Reply:
It seems like some commits from a different branch bled into this PR.